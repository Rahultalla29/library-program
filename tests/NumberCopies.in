NUMBER COPIES
ADD COLLECTION sample2.csv
NUMBER COPIES 
EXIT
