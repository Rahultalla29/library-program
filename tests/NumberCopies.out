user: No books in system.

user: 17 books successfully added.

user: Batman Volume 1 (Bob Kane): 1
Biggles in the Battle (W.E. Johns): 2
Halloween (Jamie Lee Curtis): 1
Harry Potter 1 (J.K. Rowling): 6
So Long and Thanks for all the Fish (Douglas Adams): 1
The Hitchhiker's Guide to the Galaxy (Douglas Adams): 1
The Iliad (Homer): 2
To Kill a Mockingbird (Harper Lee): 2
Wasteland (T.S. Eliot): 1

user: Ending Library process.
